diff --git a/openrewrite/src/main/java/com/example/demo/SampleController.java b/openrewrite/src/main/java/com/example/demo/SampleController.java
index e867967..d89c8cc 100644
--- a/openrewrite/src/main/java/com/example/demo/SampleController.java
+++ b/openrewrite/src/main/java/com/example/demo/SampleController.java
@@ -1,12 +1,12 @@ org.openrewrite.java.ChangeType, org.openrewrite.java.ChangePackage
 package com.example.demo;
 
+import jakarta.annotation.PostConstruct;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import javax.annotation.PostConstruct;
-import javax.validation.constraints.Max;
+import jakarta.validation.constraints.Max;
 
 @Validated
 @RestController

diff --git a/openrewrite/src/main/resources/application.properties b/openrewrite/src/main/resources/application.properties
index 8b13789..d3e825f 100644
--- a/openrewrite/src/main/resources/application.properties
+++ b/openrewrite/src/main/resources/application.properties
@@ -1 +1,5 @@ org.openrewrite.java.spring.AddSpringProperty
 
+management.endpoints.jmx.exposure.include=*
+management.endpoint.configprops.show-values=ALWAYS
+management.endpoint.env.show-values=ALWAYS
+management.endpoint.quartz.show-values=ALWAYS

diff --git a/openrewrite/build.gradle b/openrewrite/build.gradle
index 1da7977..263fe6a 100644
--- a/openrewrite/build.gradle
+++ b/openrewrite/build.gradle
@@ -1,13 +1,13 @@ org.openrewrite.gradle.ChangeJavaCompatibility, org.openrewrite.gradle.plugins.UpgradePluginVersion
 plugins {
 	id 'java'
-	id 'org.springframework.boot' version '2.7.6'
-	id 'io.spring.dependency-management' version '1.0.15.RELEASE'
+	id 'org.springframework.boot' version '3.0.0'
+	id 'io.spring.dependency-management' version '1.1.0'
 	id("org.openrewrite.rewrite") version("5.32.0")
 }
 
 group = 'com.example'
 version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '11'
+sourceCompatibility = 17
 
 repositories {
 	mavenCentral()

